File: PHYTHON/DAY2/third_task.py
def get_amount_likes(names_list):
	if type(names_list) != list:
		raise TypeError
		feed_back = ""
	if len(names_list) == 0 : feed_back = 'No One Like This'
	if len(names_list) == 1 : feed_back = str(names_list[0]) + " Like This "
	if len(names_list) == 2 : feed_back = str(names_list[0]) + ' and ' + str(names_list[1]) + ' Likes This'
	if len(names_list) == 3 : feed_back = str(names_list[0]) + ',' + str(names_list[1]) + ' and ' + str(names_list[1]) + ' Likes This'
	if len(names_list) > 3: feed_back =  str(names_list[0]) + ',' + str(names_list[1]) + ' and ' + str(len(names_list) - 2) + ' Likes This'
	return feed_back
	

File: JAVASCRIPT/Day2/third_task.js
function getAmountPeopleLikes(array){
	let feedback
	if(array.length == 0) feedback = 'No One Likes This'
	if(array.length == 1) feedback = `${array[0]} Likes This`
	if(array.length == 2) feedback = `${array[0]} and ${array[1]} Likes This`
	if(array.length == 3) feedback = `${array[0]},${array[1]} and ${array[2]} Likes This`
	if(array.length > 3) feedback = `${array[0]},${array[1]} and ${array.length - 2} Likes This`	
	return feedback
	}
console.log(getAmountPeopleLikes(['Ade','Qudus','Juwon']))
console.log(getAmountPeopleLikes(['Ade','Qudus','Juwonn','Fatai','Yemi','Lattef']))